### PR TITLE
ATA2-707 cfg: remove URL-related conditional statement as the env var is always provided

### PR DESCRIPTION
**Pull request by kubaprzetakiewicz**
## Jira
https://plentific.atlassian.net/browse/ATA2-707

### CODE REVIEW

File: .github/workflows/build_py.yml [hunk 1]
```diff
@@ -15,15 +15,13 @@ on:
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
-
+    runs-on: ubuntu-20.04
```
Comment (kubaprzetakiewicz) [inline .github/workflows/build_py.yml:18]: https://github.com/actions/setup-python/issues/544#issuecomment-1332535877

**Review by msobas — DISMISSED** [review]

**Comment by msobas** [comment]
but let's QA this with `payments` :)

**Review by klplentific — DISMISSED** [review]

File: mangopay/__init__.py [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
-
-if mangopay_url_env_var is not None:
-    api_url = f'{mangopay_url_env_var}/v2.01/'
-    api_sandbox_url = f'{mangopay_url_env_var}/v2.01/'
-else:
-    api_url = 'https://api.mangopay.com/v2.01/'
-    api_sandbox_url = 'https://api.sandbox.mangopay.com/v2.01/'
+
+MANGOPAY_IS_MOCK_SERVER_ENABLED = os.getenv("MANGOPAY_IS_MOCK_SERVER_ENABLED", "")
+MANGOPAY_MOCK_SERVER_URL = os.getenv("MANGOPAY_MOCK_SERVER_URL", "")
```
Comment (rostislav-simonik-plc) [inline mangopay/__init__.py]: can we provide default value to fallback to the local environment address, so we don't have to specify such environment variable for local environment?

Comment (rostislav-simonik-plc) [reply mangopay/__init__.py]: Not relevant anymore, we will do it in `payments/settings/local.py`

**Review by rostislav-simonik-plc — DISMISSED** [review]

**Review by msobas — APPROVED** [review]

**Pull request merged**